resources/views/templates/default.blade.php

```diff
@@ -255,9 +263,11 @@
             </p>
         @endif
 
-        <p>
-            {{ trans('invoices::invoice.amount_in_words') }}: {{ $invoice->getTotalAmountInWords() }}
-        </p>
+        @if(App::isLocale('en'))
```

Comment:
if you need this specific logic please make your own template, this is why we support templating and invoice isn't necessarily same locale as your application

https://github.com/LaravelDaily/laravel-invoices/issues/30